Svenbjorn/visitfaroeislands.com
# fixed various issues on the main menu

How to test:
- Check that the images on the hovermenu for "See & do" have same spacing horizontally and vertically
- Check that widgets on hover menu are placed and sized properly on all screen sizes
- All the onHover menus should have their content spaced equally on the top and bottom (bottom padding was too small)
- My css should not have created conflicts anywhere else, but I am not able to 100% verify this
